### DISCUSSION TITLE
Need help with sbt formatting

### DISCUSSION THREAD

**Discussion by san-coding** [http4s/http4s#5223] [Q&A]
#### I feel that scastie online formatter should work if given the right configuration, can anyone help me out with what configurations are to be given @isomarcte @hamnis @rossabaker 
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/65719940/133918093-b6e4e4ac-2c7d-48c0-835d-d2005f8ce40d.png">

**Comment by armanbilge** [comment]
I don't think scastie is powerful enough for development for a project like http4s. Have you tried VSCode in gitpod? https://gitpod.io/#https://github.com/http4s/http4s/

**Reply by san-coding** [reply]
Haven't tried it yet, will try
